website/requirements.txt

```diff
@@ -1,6 +1,7 @@
 Flask==2.1.2
 firebase-admin==5.2.0
 gunicorn==20.1.0
+importlib-metadata==5.1.0
```

Comment:
what's up with this new addition?

Reply:
The lack of this module caused a [build failure](https://console.cloud.google.com/cloud-build/builds;region=global/107a22e8-9e46-42f1-966a-5cd8047af6f8?authuser=1&project=anassri-emblem-joonix1):
```
> emblem-e2e-tests@0.0.1 test
> playwright test

Traceback (most recent call last):
  File "/usr/lib/python3.8/runpy.py", line 185, in _run_module_as_main
    mod_name, mod_spec, code = _get_module_details(mod_name, _Error)
  File "/usr/lib/python3.8/runpy.py", line 144, in _get_module_details
    return _get_module_details(pkg_main_name, error)
  File "/usr/lib/python3.8/runpy.py", line 111, in _get_module_details
    __import__(pkg_name)
  File "/workspace/website/flask/__init__.py", line 7, in <module>
    from .app import Flask as Flask
  File "/workspace/website/flask/app.py", line 27, in <module>
    from . import cli
  File "/workspace/website/flask/cli.py", line 39, in <module>
    import importlib_metadata as metadata  # type: ignore
ModuleNotFoundError: No module named 'importlib_metadata'
```

Reply:
ack